cupyx/scipy/sparse/linalg/_eigen.py

```diff
@@ -319,17 +324,41 @@ def svds(a, k=6, *, ncv=None, tol=0, which='LM', maxiter=None,
         raise ValueError('k must be smaller than min(m, n) (actual: {})'
                          ''.format(k))
 
-    aH = a.conj().T
```

Comment:
How about changing this line simply?
```py
aH = a.H if isinstance(a, _interface.LinearOperator) else a.conj().T
```

Reply:
How about the following implementation for the simplification?
```py
a = _interface.aslinearoperator(a)
if m >= n:
    a, aH = a, a.H
else:
    a, aH = a.H, a
...
w, x = eigsh(aH @ a, ...)
```

Reply:
Thanks! Certainly, that is simpler.